client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/index.tsx

```diff
@@ -44,8 +47,23 @@ const Launchpad: Step = ( { navigation, flow }: LaunchpadProps ) => {
 		[]
 	);
 
-	if ( ! isLoggedIn ) {
+	const redirectToSiteHome = useCallback( ( siteSlug: string | null, flow: string | null ) => {
+		recordTracksEvent( 'calypso_launchpad_redirect_to_home', { flow: flow } );
 		window.location.replace( `/home/${ siteSlug }` );
+	}, [] );
+
```

Comment:
Adding redirectToSiteHome() to reduce duplication and ensure consistency. We hadn't been logging the calypso_launchpad_redirect_to_home event when redirecting on line 47 for the isLoggedIn check.